### PR TITLE
Temorarily use dev activities api as dummy

### PR DESCRIPTION
**Pull request by steverendell**

### CODE REVIEW

File: helm_deploy/values-dev.yaml [hunk 1]
```diff
@@ -20,5 +20,11 @@ generic-prometheus-alerts:
   sqsAlertsQueueNames:
     - "Digital-Prison-Services-dev-hmpps_prisoner_to_nomis_queue"
     - "Digital-Prison-Services-dev-hmpps_prisoner_to_nomis_dlq"
+    - "Digital-Prison-Services-dev-hmpps_prisoner_to_nomis_visit_queue"
+    - "Digital-Prison-Services-dev-hmpps_prisoner_to_nomis_visit_dlq"
+    - "Digital-Prison-Services-dev-hmpps_prisoner_to_nomis_incentive_queue"
+    - "Digital-Prison-Services-dev-hmpps_prisoner_to_nomis_incentive_dlq"
+    - "Digital-Prison-Services-dev-hmpps_prisoner_to_nomis_activity_queue"
+    - "Digital-Prison-Services-dev-hmpps_prisoner_to_nomis_activity_dlq"
```
Comment (steverendell) [inline helm_deploy/values-dev.yaml:28]: add new queue alerts

File: helm_deploy/values-prod.yaml [hunk 2]
```diff
@@ -14,11 +14,17 @@ generic-service:
     API_BASE_URL_MAPPING: https://nomis-sync-prisoner-mapping.hmpps.service.justice.gov.uk
     API_BASE_URL_VISITS: https://visit-scheduler.prison.service.justice.gov.uk
     API_BASE_URL_INCENTIVES: https://incentives-api.hmpps.service.justice.gov.uk
-    API_BASE_URL_ACTIVITIES: https://activities-api.prison.service.justice.gov.uk
+    API_BASE_URL_ACTIVITIES: https://activities-api-dev.prison.service.justice.gov.uk
```
Comment (andymarke) [inline helm_deploy/values-prod.yaml:17]: this is fine, but recently we just point back at this service, it just needs to have a health point

**Review by andymarke — APPROVED** [review]
lgtm

File: helm_deploy/values-prod.yaml (hunk 2 above)
Comment (steverendell) [reply helm_deploy/values-prod.yaml:17]: ok noted!

**Pull request merged**